### ISSUE TITLE
Missing heading levels in several pages

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#539]
Home page
- [x] needs an sr-only h2 for the features section:  "Features"

Selections
- [x] sr-only h1:  "Selections list"
- [x] page title: "Selections list in SearchWorks"

Advanced search
- [x] Match all/any; Limit results by; Sort by should all be h2.
- [x] remove the "Clear form" button (should def not be part of h1; and I don't think it's required at all unless we are retaining form content)

Search results pages without a masthead
- [x] sr-only h1 "Search results" 

No results
- [x] h1: "No results"
- [x] needs an sr-only h2 above the left sidebar: "Assistance"

ALL SEARCH RESULTS PAGES
- [x] there is a redundant h2 after making the number of results an h2... please delete the original one (or merge them or whatever):

```
<h2 class="sr-only top-content-title">Search</h2>
```
